iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -149,6 +164,27 @@ struct LinalgVectorLoweringPass
 
   void runOnOperation() override;
 };
+
+struct UnrollOneVectorOpPass
```

Comment:
Do you want to also add the options struct for these?

Reply:
Yes, I do. I'd like to add them in another PR, so we can make this PR cleaner and simpler, i.e., all the codes are copied from sandbox without modification.